docker-for-mac/release-notes.md

```diff
@@ -25,6 +25,32 @@ This page contains information about the new features, improvements, known issue
 
 **Compose V2 beta**: Docker Desktop now includes the beta version of Compose V2, which supports the `docker compose` command as part of the Docker CLI. For more information, see [Compose V2 beta](../compose/cli-command.md). While `docker-compose` is still supported and maintained, Compose V2 implementation relies directly on the compose-go bindings which are maintained as part of the specification. The compose command in the Docker CLI supports most of the `docker-compose` commands and flags. It is expected to be a drop-in replacement for `docker-compose`. There are a few remaining flags that have yet to be implemented, see the [docker-compose compatibility list](../compose/cli-command-compatibility.md) for more information about the flags that are supported in the new compose command. If you run into any problems with Compose V2, you can easily switch back to Compose v1 by either by making changes in Docker Desktop **Experimental** Settings, or by running the command `docker-compose disable-v2`. Let us know your feedback on the new ‘compose’ command by creating an issue in the [Compose-CLI](https://github.com/docker/compose-cli/issues) GitHub repository.
 
+### Upgrades
+
+- [Compose V2 beta](https://github.com/docker/compose-cli/releases/tag/v2.0.0-beta.5)
+  - `compose run` and `compose exec` use separate streams for stdout and stderr
+  - `compose run` and `compose exec` support detach keys
+  - fixed `--force` and `--volumes` flags on compose rm command
+  - fixed network's IPAM configuration. Service can define a fixed IP
+
+### Bug fixes and minor changes
+
+- Fix connection leak when using `virtualization.framework` and not using `vpnkit`.
+- Fix spurious traces on iptables updates.
+- Fix slowness when adding multiple ports forwarding option.
+
```

Comment:
- Dev Environments: Fix blank screen in some create/remove scenarios. Fixes https://github.com/docker/dev-environments/issues/4
- Dev Environments: Fix error handling in removing an environment. Fixes https://github.com/docker/dev-environments/issues/8
- Dev Environments: Disable Start/Stop/Share buttons while an environment is being created or removed

Reply:
Added in both mac and win!